.github/workflows/release_library.yml

```diff
@@ -187,6 +189,17 @@ jobs:
         env:
           GH_TOKEN: ${{ steps.get-workflow-token.outputs.token }}
 
+  check-dependencies:
+    name: check-dependencies
+    uses: reside-eng/workflow-templates/.github/workflows/check-nodejs-dependencies.yml@v10
+    with:
+      WORKFLOW_NAME: check-dependencies
+      ENV: stage
+      GIT_REF: ''
+    secrets:
+      NPM_READ_TOKEN: ${{ secrets.NPM_READ_TOKEN }}
```

Comment:
We should add the `NPM_READ_TOKEN` new required secret to this workflow definition for better clarity.